cabby/geo/walk_test.py

```diff
@@ -17,37 +17,53 @@
 import unittest
 from shapely.geometry.point import Point
 from cabby.geo import walk
+from cabby.geo.map_processing import map_structure
 
 
 class WalkTest(unittest.TestCase):
 
     def setUp(self):
-        
+
         # Load map from disk.
-        self.map = map_structure.Map("Manhattan", 18, "/mnt/hackney/data/cabby/poi/")
+        self.map = map_structure.Map(
+            "Manhattan", 18, "/mnt/hackney/data/cabby/poi/v1/")
```

Comment:
Shouldn't reference files on disk, please switch this to Bologna (as our test case for things like this).